src/osd/PeeringState.h

```diff
@@ -655,6 +655,7 @@ class PeeringState : public MissingLoc::MappingInfo {
   //       GetMissing
   //       WaitUpThru
   //       Incomplete
+  //       Down
```

Comment:
?

Reply:
Might be another PeeringState we are missing @shraddhaag, let us know if this is what you intended.

Reply:
I hadn't added that, it was part of the prior art. Have removed it since it doesn't seem related!